workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -65,13 +65,15 @@ private void GenerateComponentGroups()
                     GetComponentGroup(gameObjectComponentDispatcher.ComponentAddedComponentTypes);
                 gameObjectComponentDispatcher.ComponentRemovedComponentGroup =
                     GetComponentGroup(gameObjectComponentDispatcher.ComponentRemovedComponentTypes);
-                gameObjectComponentDispatcher.AuthoritiesChangedComponentGroup =
-                    GetComponentGroup(gameObjectComponentDispatcher.AuthoritiesChangedComponentTypes);
-                if (gameObjectComponentDispatcher.ComponentsUpdatedComponentTypes.Length > 0)
-                {
-                    gameObjectComponentDispatcher.ComponentsUpdatedComponentGroup =
-                        GetComponentGroup(gameObjectComponentDispatcher.ComponentsUpdatedComponentTypes);
-                }
+                gameObjectComponentDispatcher.AuthorityGainedComponentGroup =
+                    GetComponentGroup(gameObjectComponentDispatcher.AuthorityGainedComponentTypes);
+                gameObjectComponentDispatcher.AuthorityLostComponentGroup =
+                    GetComponentGroup(gameObjectComponentDispatcher.AuthorityLostComponentTypes);
+                gameObjectComponentDispatcher.AuthorityLossImminentComponentGroup =
+                    GetComponentGroup(gameObjectComponentDispatcher.AuthorityLossImminentComponentTypes);
+                gameObjectComponentDispatcher.ComponentsUpdatedComponentGroup =
```

Comment:
It looks like the previous `if (gameObjectComponentDispatcher.ComponentsUpdatedComponentTypes.Length > 0)` part has been removed. Is that intentional?

Reply:
Not sure why I did that, reverting